portfolio/src/main/webapp/script.js

```diff
@@ -38,3 +38,33 @@ function addRandomStock() {
   const stockContainer = document.getElementById('stock-container');
   stockContainer.innerText = stock;
 }
+
+// Fetches comments from the server and adds them to the DOM.
+function LoadComments(){
+  var max=getMax();
```

Comment:
Here and throughout this file, let's make sure we have a spaces before and after equal signs. So `var max = getMax()` instead of `var max=getMax()`.

Reply:
Addressed in latest commit.